circuits/src/keccak/multivar.rs

```diff
@@ -147,23 +160,27 @@ impl<F: EccPrimeField<Repr = [u8; 32]>> KeccakMultiVarHasher<F> {
         // preimage_var now holds the desired non-native limbs. Form preimage
         // as the fixed-length input followed by the bytes in the non-native limbs.
         let mut preimage = Vec::with_capacity(
-            self.fixed_input.len() + var_preimage_len / 3 * 32,
+            self.fixed_input.len()
+                + var_preimage_len / NUM_LIMBS * NUM_BYTES_FQ,
```

Comment:
perhaps add a comment saying that `var_preimage_len` is measured in limbs, so to get the bytes you need to compute the number of Fq elements and then multiply by NUM_BYTES_FQ

Reply:
Updated the comment.